src/ddprof_cmdline.cc

```diff
@@ -224,30 +240,33 @@ bool watcher_from_tracepoint(const char *_str, PerfWatcher *watcher) {
   }
 
   // Check enablement, just to print a log.  We still enable instrumentation.
-  snprintf(path, sizeof(path), "/sys/kernel/tracing/events/%s/%s/enable",
-           groupname, tracename);
+  snprintf(path, sz_path, "%s/%s/%s/enable", spath, gname, tname);
   fd = open(path, O_RDONLY);
   if (-1 == fd || 1 != read(fd, buf, 1) || '0' != *buf) {
     LG_NTC("Tracepint %s:%s is not enabled.  Instrumentation will proceed, but "
            "you may not have any events.",
-           groupname, tracename);
+           gname, tname);
   } else {
-    LG_NFO("Tracepoint %s:%s successfully enabled", groupname, tracename);
+    LG_NFO("Tracepoint %s:%s successfully enabled", gname, tname);
   }
 
   // OK done
   *watcher = *twatcher_default();
   watcher->config = trace_id;
   watcher->sample_period = period;
-  if (is_raw)
-    watcher->sample_type |= PERF_SAMPLE_RAW;
+
+  // das221004 the raw field is a very small part of the overall event and
```

Comment:
This is unfortunate, but I didn't want to fix it in this PR.  Should only affect tracepoints.